Wiktionary_scraper.py

```diff
@@ -135,7 +147,14 @@ def write_line_by_line(words, filename):
 
     for word in new_words:
         with open(filename, 'a+', encoding="utf-8") as file:
-            prons = pronunciations_from_wiktionary(word)
+            if language == 'english':
```

Comment:
it would be more elegant to have a dictionary keyed by language name and which values are the function.
```
wiktionaryPronounciations = {
    'italian': pronunciations_from_wiktionary_italian,
    'french': pronunciations_from_wiktionary_french,
    'english': pronunciations_from_wiktionary 
}
fn = wiktionaryPronounciations.get(language, None)
if fn:
   prons = fn(word)
```

This could be encapsulated in a class too...